hamberger.js

```diff
@@ -0,0 +1,13 @@
+const option = document.querySelector('.option');
+const menu = document.querySelector('.hamberger');
+const closemenu = document.querySelector('.Closebtn');
+
+menu.addEventListener('click', () => {
+  option.style.display = 'flex';
+  menu.style.display = 'block';
+});
+
+closemenu.addEventListener('click', () => {
+  option.style.top = '-100%';
+  closemenu.style.left = '0';
+});
```

Comment:
- [ ] Please 🙏🏽 complete the work section dynamically and implement the popup window for both sizes mobile and desktop versions. By creating this section dynamically adding projects in the future will be easy.